### DISCUSSION TITLE
multi import problem

### DISCUSSION THREAD

**Discussion by MirS0bhan** [fastapi/sqlmodel#1141] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class User(SQLModel, table=True):
    __tablename__ = 'users1'
    id: int = Field(default=None, primary_key=True)
    username: str = Field(index=True, unique=True, max_length=50)
    email: str = Field(index=True, unique=True, max_length=100)
    hashed_password: str
    created_at: datetime = Field(default_factory=datetime.utcnow)

    cards: List["Card"] = Relationship(back_populates="owner")
    decks: List["Deck"] = Relationship(back_populates="owner")
```

### Description

when i imported this model in two files i got this error : `sqlalchemy.exc.InvalidRequestError: Table 'users1' is already defined for this MetaData instance.  Specify 'extend_existing=True' to redefine options and columns on an existing Table object.
`

### Operating System

Linux

### SQLModel Version

0.0.22

### Python Version

Python 3.12.6

**Comment by YuriiMotov** [comment]
Could you provide an [MRE](https://stackoverflow.com/help/minimal-reproducible-example)?

**Reply by MirS0bhan** [reply]
sorry I'm not working on it and I forgot the context
